.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+Please use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) for usage, installation, or modeling questions, or other requests for help.
+_Do not post such requests to Issues._ Doing so interferes with the development of Caffe.
+
+Please read the [guidelines for contributing](https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md) before submitting this issue.
+
+### Your system configuration
+Operating system:
+Compiler:
+Python version:
```

Comment:
Should include "(if applicable)" here too, since pycaffe is common but not ubiquitous, or consider an alternative line like "Python or MATLAB version (for pycaffe and matcaffe respectively)"